clients/kotlin/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,151 @@
+import features.config.*
+import features.invoke.*
+import features.resolve.*
+import features.subinvoke.*
+import features.uri.*
+import features.wrapFeature.*
+import features.wrapType.*
+import util.*
+
+fun main(args: Array<String>) {
+    val filter: String? = if (args.isNotEmpty() && args[0].isNotEmpty()) args[0] else null
+    val loader = SpecReader(root().resolve("specs"))
+
+    val specs: List<String> = listOf(
+        "config_embed_wrap_package",
+        "config_env_variables",
+        "config_interface_implementations",
+        "config_plugin_instance",
+        "config_plugin_package",
+//        "config_resolver",
```

Comment:
Any tests that are skipped should be marked with a :x: in the root README

Reply:
Doesn't the ❌ imply the client can't do it? But really the client can add resolvers and can resolve URIs for invocations. Should we maybe have another shape to indicate this?

Reply:
I added the :x: next to all the skipped tests. The Kotlin client does support the config and URI resolution features that aren't being tested. It just doesn't support `tryResolveUri` or `loadWrapper`, which are required to complete the tests. 

If we could, for example, prove IPFS resolution by invoking a wrapper via IPFS, the Kotlin client would pass the test. I believe the same is true for all of the skipped tests except ENS resolution, which will be supported once we merge  https://github.com/polywrap/rust-client/pull/165

Reply:
Okay makes sense @krisbitney. Yah I think an orange circle 🟠 could work well for this. I’d also say we should update the test specs to not require tryResolveUri